src/components/sitepages/SiteTargets.vue

```diff
@@ -73,7 +73,7 @@
                         class="sidebar-tab-button">telescope controls</div>
                     <div 
                         :class="{'active': activeSidebarTab=='common targets'}" 
-                        @click="activeSidebarTab='common targets'; submitForm()"
+                        @click="activeSidebarTab='common targets'; submitForm(); show_common_targets=true"
```

Comment:
I think this would be cleaner if you define a click handler method instead of having multiple statements in the html section

Reply:
Cleaned it up as best I can - submitForm is used elsewhere and would be a lot of code to copy into a handler as well, but let me know if you had something different in mind